README.md

```diff
@@ -1,26 +1,14 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# Overview
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+This charm deploys a reverse proxy in front of other services. You can use this to load balance existing deployments.
 
-Use links instead.
--->
+# Usage
 
-# is-charms-template
+```
+juju deploy haproxy
```

Comment:
Don't you need to relate it to another charm in order to front the later?

Reply:
For now the charm doesn't do much so I'm thinking to keep it like this and update it once I have the integrations implemented.